src/Microsoft.DotNet.Tools.New/Program.cs

```diff
@@ -79,8 +77,27 @@ public static int Main(string[] args)
             app.Description = "Initializes empty project for .NET Platform";
             app.HelpOption("-h|--help");
 
+            var lang = app.Option("-l|--lang <LANGUAGE>", "Language of project ( C#, F# )", CommandOptionType.SingleValue);
```

Comment:
Yeah. I'm thinking, should we remove the "c#" option and just leave "csharp" and "cs" (same for F#)? makes more snese...